go/client/cmd_wait.go

```diff
@@ -0,0 +1,93 @@
+package client
+
+import (
+	"fmt"
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+	"golang.org/x/net/context"
+	"time"
+)
+
+type CmdWait struct {
+	libkb.Contextified
+	duration time.Duration
+}
+
+const maxWaitTime = 60
+
+func NewCmdWait(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "wait",
+		Usage: "Waits for the keybase server to start up",
+		Action: func(c *cli.Context) {
+			cl.SetForkCmd(libcmdline.NoFork)
+			cl.SetLogForward(libcmdline.LogForwardNone)
+			cl.ChooseCommand(&CmdWait{Contextified: libkb.NewContextified(g)}, "wait", c)
+		},
+		Flags: []cli.Flag{
+			cli.DurationFlag{
+				Name:   "duration, d",
+				Value: 10 * time.Second,
+				Usage:  "How long to wait before timing out",
+			},
+		},
+	}
+}
+
+func (c *CmdWait) ParseArgv(ctx *cli.Context) error {
+	if len(ctx.Args()) > 0 {
+		return UnexpectedArgsError("wait")
+	}
+
+	c.duration = ctx.Duration("duration")
+	if c.duration.Seconds() <= 0 || c.duration.Seconds() > maxWaitTime {
+		return fmt.Errorf("invalid duration %s, must be between 1s and %ds", c.duration, maxWaitTime)
```

Comment:
```suggestion
		return fmt.Errorf("invalid duration %s, must be between 1s and %s", c.duration, maxWaitTime)
```

Reply:
But then it looks like `invalid duration 9m15s, must be between 1s and 60`
no units on `60`